### PR TITLE
Don't use global namespaced Contao classes

### PR DESCRIPTION
**Pull request by bytehead**
Trying to get rid of Contao's global namespaced classes.

When using `\Haste\Input\Input::getAutoItem(...)` for example PHPStan will report following:

```
Class Input not found.                                                  
💡 Learn more at https://phpstan.org/user-guide/discovering-symbols
```

PS: There is a workaround - while bootstrapping PHPStan you can boot the Contao Framework, but that's a bit overkill IMO.

### CODE REVIEW

**Comment by qzminski** [comment]
@Toflar @aschempp should be safe to merge, or do you see any potential issues with this?

**Review by aschempp — APPROVED** [review]

File: library/Haste/Generator/ModelData.php [hunk 1]
```diff
@@ -56,7 +59,7 @@ public static function generate(\Model $objModel=null, $varCallable=null)
 
                 if ($objRelated == null) {
                     $arrData[$strField] = new Plain('', $strLabel, $arrAdditional);
-                } elseif ($objRelated instanceof \Model\Collection) {
+                } elseif ($objRelated instanceof Model\Collection) {
```
Comment (bytehead) [inline library/Haste/Generator/ModelData.php:62]: Not sure, if you like an alias like `use Contao\Model\Collection as ContaoModelCollection` more than this how it is now?

Comment (qzminski) [reply library/Haste/Generator/ModelData.php:62]: I like the current state of things better, never been a fan of aliasing 😉

Comment (bytehead) [reply library/Haste/Generator/ModelData.php:62]: Fine for me. :)

**Comment by qzminski** [comment]
Just released as 4.25, thank you @bytehead !

**Comment by bytehead** [comment]
Thanks for merging! 🙏  You're welcome!

**Pull request merged**